.github/workflows/build.yml

```diff
@@ -44,7 +44,7 @@ jobs:
   # job 2: Test on 'arm64v8/ubuntu' OS.
   build_arm64-binary:
     runs-on: ubuntu-latest
-    if: ${{ github.event_name == 'push' && (github.ref_name == 'develop' || github.ref_name == 'snapshot' || github.ref_name == '2.x') }}
+#    if: ${{ github.event_name == 'push' && (github.ref_name == 'develop' || github.ref_name == 'snapshot' || github.ref_name == '2.x') }}
```

Comment:
This should not be commented out here, we need ci build

Reply:
> This should not be commented out here, we need ci build

Here should be just to test arm build only, because in the previous community tried to upgrade, encountered a high version of spring boot can not be compiled on the ci, but arm local compilation is not a problem
这里应该只是为了测试arm build而已,因为在之前社区尝试过升级,遇到过高版本spring boot无法在ci上通过编译,但是arm本地编译没有问题

Reply:
https://github.com/seata/seata/issues/5808#issuecomment-1773772194

Reply:
> > This should not be commented out here, we need ci build
> 
> Here should be just to test arm build only, because in the previous community tried to upgrade, encountered a high version of spring boot can not be compiled on the ci, but arm local compilation is not a problem 这里应该只是为了测试arm build而已,因为在之前社区尝试过升级,遇到过高版本spring boot无法在ci上通过编译,但是arm本地编译没有问题

ok，I see